site/ESRs/esr1.qmd

```diff
@@ -8,26 +8,25 @@ title: "ESR1: Albert Navarro Gallinad"
 
 ![](../assets/esr_pictures/albert_square.png){.img-profile}
 
-{{< fa brands github >}} \ [navarral](https://github.com/navarral)<br>
-{{< fa brands linkedin >}}\  [LinkedIn](https://www.linkedin.com/in/alfontal/)<br>
+{{< fa brands github >}} \ [navarral](https://navarral.github.io/)<br>
```

Comment:
May I suggest to keep both your main site and github links separately, as they serve different purposes?
It would look like it does on [my profile](https://helical-itn.eu/ESRs/esr6.html). You would just need to add this line:
```suggestion
{{< fa brands github >}} \ [navarral](https://github.com/navarral)<br>
{{< fa brands globe >}} \ [navarral.github.io](https://navarral.github.io/)<br>
```

Reply:
BTW: you can accept the suggestion and commit directly from the button above 😄
<img width="208" alt="imagen" src="https://user-images.githubusercontent.com/10958332/209164423-2c49d386-13c9-406e-ae24-b0635d246009.png">